anyric/super-record
# #167260746 Add Deactivate and Activate functionalities

#### What does this PR do?
This PR add implementation for deactivating and activating a user by an admin

#### Description of Task to be completed?
- Add a modal form for deactivating users

#### How should this be manually tested?
- Clone this branch
- In the root directory run `python manage.py runserver`
- Login to the system
- Navigate to `http://127.0.0.1:5000/setting` and select user menu
- Click on the `deactivate` or `activate` action switches
- Confirm the action from the modal form to complete

#### Any background context you want to provide?
Currently there is no provision for admins to deactivate users in the system. There should be a provision for deactivating users.

#### What are the relevant pivotal tracker stories?
[#167260746](https://www.pivotaltracker.com/story/show/167260746)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29800415/61523027-36ff0680-aa1c-11e9-8004-09bbcb7dc0e7.png)

![image](https://user-images.githubusercontent.com/29800415/61523056-441bf580-aa1c-11e9-8613-78a44c999054.png)

![image](https://user-images.githubusercontent.com/29800415/61523084-5138e480-aa1c-11e9-9a12-2f32e478fb37.png)

#### Questions:
N/A
